src/spawn.c

```diff
@@ -67,6 +67,7 @@
 #else
 # include "support.h"
 #endif
+#include "utils.h"
```

Comment:
indeed, but actually it should really be locale→UTF-8 conversion as mentioned below.  Or remove it altogether and don't allow locale data, and hope we don't break API.  Or just break API and officially expect UTF-8 on Windows.